region/norfair/crocomire.json

```diff
@@ -2318,18 +2530,64 @@
               ]
             },
             {
-              "id": 2,
+              "id": 3,
               "strats": [
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": [],
-                  "obstacles": [
-                    {
-                      "id": "A",
-                      "requires": [ "h_canOpenGreenDoors" ]
-                    }
+                  "requires": [
+                    {"or":[
+                      "Grapple",
+                      "SpaceJump"
+                    ]}
                   ]
+                },
+                {
+                  "name": "Gravity",
+                  "notable": false,
+                  "requires": [
+                    "Gravity",
+                    {"or": [
+                      "canWalljump",
+                      {"and": [
+                        "HiJump",
+                        {"or": [
+                          "canTrickyJump",
+                          "canUseFrozenEnemies",
+                          "canTrickyDashJump"
```

Comment:
`canTrickyDashJump` requires `canTrickyJump`. I wouldnt include the speed version as well, unless it's a fair bit easier

Reply:
Is the tricky jump alone even possible?

Reply:
It doesn't look possible.  Seems likely I had speed on at the time.  And the speed+HiJump jump from the stairs is much easier than running along the bottom.

Reply:
And for `FrozenEnemies` it needs either more beam damage or crouchjumpdowngrab